web/test/integration/quickstart/quickstart.feature

```diff
@@ -0,0 +1,16 @@
+Feature:  Display quickstart on helppage 
+
+Scenario: List quickstart on helppage
+    Given the "/help" page is open
+    And the text "Quickstart" is in the element ".help__card__content__title"
+    When enter text "Glossary" in the element ".help__card__header__search.mat-input-element"
+    And the ".help__card__content__title" link at position 0 is clicked
+    Then the text "Get up and running in minutes" is in the element "#get-up-and-running-in-minutes-in-3-simple-steps-and-without-installing-anything"
+
+Scenario: Ensure search in quickstart on helppage
```

Comment:
If this test fails the `Scenario` is not clear what is being tested. I will update